File: gdx2d-demoDesktop/src/ch/hevs/gdx2d/demos/physics/pinball/Sensor.java
package ch.hevs.gdx2d.demos.physics.pinball;

import ch.hevs.gdx2d.components.physics.primitives.PhysicsStaticBox;
import com.badlogic.gdx.math.Vector2;



public class Sensor extends PhysicsStaticBox {

	public Sensor(String name, Vector2 position, float width, float height) {
		super(name, position, width, height);
		setSensor(true);
	}
}

File: gdx2d-demoDesktop/src/main/scala/ch/hevs/gdx2d/demos/physics/pinball/Sensor.scala
package ch.hevs.gdx2d.demos.physics.pinball

import ch.hevs.gdx2d.components.physics.primitives.PhysicsStaticBox
import com.badlogic.gdx.math.Vector2

class Sensor(name: String, position: Vector2, width: Float, height: Float) extends PhysicsStaticBox(name, position, width, height) {
  setSensor(true)
}
